src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -191,11 +194,6 @@ remove all whitespace including the final carriage return.
 For the `-key` flag use the key created earlier and stored in the media.key
 file. However, when entering it on the command line, be sure you've removed its
 whitespace. For the `-key_id` flag repeat the key value.
```

Comment:
`key_id` is really an identifier that uniquely identifies the `key`. It can usually be found in the manifests and media containers in plain text/hex.

Some applications concatenate `key_id` and `key` and store them together, e.g. Apple's segmenter tool. This gives an impression that `key_id` is part of the `key`, but it is actually not.

The size of key needs to be 16 bytes because we are using AES-128 (128 bits, 16 bytes) for CENC and SAMPLE-AES.

The requirement of key-id being 16 bytes comes from CENC spec (23001-7) which requires key-id (KID) to be 16 bytes.